Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -401,6 +401,8 @@ - (void)_installScreenTimeWebpageController
 
         [_screenTimeWebpageController setSuppressUsageRecording:![_configuration websiteDataStore].isPersistent];
 
+        [_screenTimeWebpageController setURLIsPlayingVideo:NO];
+        [_screenTimeWebpageController setURLIsPictureInPicture:NO];
```

Comment:
Unnecessary, as the default values are `NO`, and `_screenTimeWebpageController` is instantiated directly above.